content/docs/start/data-and-model-versioning.md

```diff
@@ -5,15 +5,15 @@ regular Git workflow for datasets and ML models, without storing large files in
 Git.'
 ---
 
-# Get Started: Data Versioning
+# Data Versioning
 
 How cool would it be to make Git handle arbitrary large files and directories
 with the same performance as with small code files? Imagine doing a `git clone`
 and seeing data files and machine learning models in the workspace. Or switching
 to a different version of a 100Gb file in less than a second with a
 `git checkout`.
 
-The foundation of DVC consists of a few commands that you can run along with
+The foundation of DVC consists of a few commands which you can run along with
```

Comment:
again; bordering on unrestrictive -> which (and defaulting to restrictive -> personal style choice).

I used "which" to signify immediate explication - i.e. "that" could refer to "foundation" or "commands," but "which" can be thought of as a shorthand for "which in turn" which can only refer to "commands."

Reply:
I tried reading the original sentence a few times and can't perceive "that" referring to "foundation". Wouldn't "which" have the same problem though?

Anyway. Since we're leaving the other ones you changed let's also resolve here for consistency. 👍